### DISCUSSION TITLE
Party Size filter for Hero + Talent stats

### DISCUSSION THREAD

**Discussion by worstmedivh** [Heroes-Profile/heroesprofile#700] [Ideas]
Problem: Some of the highest full talent build winrates appear to be unrealistically high for most players because they feature a unique strange talent combination and may be only played by a specific player in a 5 stack or other party. The build could be uniquely powerful with a highly specific 5 stack comp, or otherwise affected by being played exclusively or primarily by a single or small number of players who may be benefitting from party advantage.

{Proposed Solution: "Party Size" filter as an additional filter on Global Talents statistics pages for Heroes, and for Global Hero Statistics page. As a solo SL player, it may be more useful to filter to stats from solo-only players than even to filter by Player Rank (or to do so in combination, if there is enough data). As I imagine it, the filter would only restrict the stats for the selected Hero, though there could be any combination of parties for their teammates or enemies in those games.

Possible alternatives: Weighting somehow by account, so that individual accounts which may contribute a majority or all of the games for a single build are underweighted in the stats. Or, talent builds used exclusively or primarily by a single account don't show up in the popular build statistics even if they would have enough games to otherwise appear.

Example: The Malthael -> Last Update SL statistics show a highly unusual build as the highest winrate popular build, and it turns out to be being played mainly by a single player in a 5 stack with Ana + Medivh.

https://www.heroesprofile.com/Global/Talents/?hero=Malthael&timeframe_type=last_update&timeframe=&type=win_rate&game_type=sl&league_tier=master&mirror=Exclude

I believe some of the QM statistics, particularly for higher player MMRs, are especially also biased toward builds that work in 5 stacks, due to the stronger statistical advantage of 5 stacks in QM vs. in SL.

**Comment by Zemill** [comment]
I will look into this...while the data is available, due to the pre-parsing we do for global pages, it is a bit more complicated

**Reply by worstmedivh** [reply]
After the site rewrite, just wondering again if this is something you would consider adding (personally my interest would primarily be to the "Hero Stats" and "Talent Stats" global pages).

**Reply by Zemill** [reply]
I looked into this briefly and am unsure if its feasible at this time.  I will continue to look into it as I find time
